pkg/exchange/bybit/exchange.go

```diff
@@ -0,0 +1,44 @@
+package bybit
+
+import (
+	"github.com/sirupsen/logrus"
+
+	"github.com/c9s/bbgo/pkg/exchange/bybit/bybitapi"
+	"github.com/c9s/bbgo/pkg/types"
+)
+
+// BIT is the platform currency of Bybit, pre-allocate static string here
+const BIT = "BIT"
+
+var log = logrus.WithFields(logrus.Fields{
+	"exchange": "bybit",
+})
+
+type Exchange struct {
+	key, secret string
+	client      *bybitapi.RestClient
+}
+
+func New(key, secret string) *Exchange {
+	client := bybitapi.NewClient()
+
+	// for public access mode
```

Comment:
`for public access mode` means we have to specify key and secret for public api?

Reply:
NO, this is due to i copy from kucoin...  i will remove that.